extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/BarrageUtil.java

```diff
@@ -207,6 +229,7 @@ public static Stream<Field> columnDefinitionsToFields(
             final Map<String, String> metadata = fieldMetadataFactory.apply(name);
 
             putMetadata(metadata, "isPartitioning", column.isPartitioning() + "");
+            putMetadata(metadata, "isSortable", String.valueOf(sortableColumns.contains(name)));
```

Comment:
This ends up being an additional `O(n^2)` check as sortableColumns is a List. I think I'd like a Set better.

Reply:
I converted it into a set but I was wondering what the difference would be. For either data type wouldn't I still be using this contains method to check if that column name is present within it